libs/feature/search/src/lib/organisations/organisations.container.component.ts

```diff
@@ -0,0 +1,53 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core'
+import { CatalogOrganisation } from '@geonetwork-ui/util/shared'
+import { Observable } from 'rxjs'
+import { filter, map } from 'rxjs/operators'
+import { SearchFacade } from '../state/search.facade'
+
+@Component({
+  selector: 'gn-ui-organisations-container',
+  templateUrl: './organisations.container.component.html',
+  styleUrls: ['./organisations.container.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class OrganisationsContainerComponent implements OnInit {
+  constructor(private searchfacade: SearchFacade) {}
+  organisations$: Observable<CatalogOrganisation[]>
+
+  ngOnInit(): void {
+    this.searchfacade.setConfigAggregations({
+      org: {
+        terms: {
+          size: 1000,
```

Comment:
do you have empty orgs ?
might need to use `exclude` if yes

Reply:
Right, I forgot about this. Don't have any empty orgs on current catalog, but will add it in case.